netty/src/test/java/io/grpc/netty/NettyServerHandlerTest.java

```diff
@@ -1315,7 +1313,9 @@ private void rapidReset(int burstSize) throws Exception {
     for (int period = 0; period < 3; period++) {
       for (int i = 0; i < burstSize; i++) {
         channelRead(headersFrame(streamId, headers));
+        channel().releaseOutbound();
```

Comment:
I have added these releaseOutbounds() here as i saw leakage error in the below 2 UTs earlier which are calling the rapidReset() method and it has outbound messages which are not released.

maxRstCount_withinLimit_succeeds()
maxRstCount_exceedsLimit_fails()

Reply:
`releaseOutbound()` is the sort of thing that only needs to be run once at the end of tests, to avoid leaks. Running it at the end of `@Before` could be okay if we don't want tests to see any queued messages, but there's no point to calling it multiple times here. Really, I'm surprised it isn't placed mostly just once in an `@After` to clean up after every test.

Reply:
Fixed

Reply:
I had allowed that based on [this](https://github.com/grpc/grpc-java/blob/4e8f7df589cbe5a192e705ab372f2b27957bf662/netty/src/test/java/io/grpc/netty/NettyServerHandlerTest.java#L875) existing call inside a loop, which gave me the impression that the queue might be getting emptied without releasing and refilled multiple times during the test but that is not the case.